src/magic.h

```diff
@@ -96,6 +96,12 @@ struct fake_spell {
     int level = 0;
     // target tripoint is source (true) or target (false)
     bool self = false;
+    // a chance to trigger the enchantment spells
+    int trigger_once_in;
```

Comment:
```suggestion
    int trigger_once_in = 1;
```
one in one is just 100%, so this is a reasonable default